README.md

```diff
@@ -39,10 +39,21 @@ Crop the image specified by the URI param. If URI points to a remote image, it w
 If the cropping process is successful, the resultant cropped image will be stored in the cache path, and the URI returned in the promise will point to the image in the cache path. Remember to delete the cropped image from the cache path when you are done with it.
 
 ```ts
-ImageEditor.cropImage(uri, cropData).then((url) => {
-  console.log('Cropped image uri', url);
-  // In case of Web, the `url` is the base64 string
-});
+ImageEditor.cropImage(uri, cropData).then(
+  ({
+    uri, // the path to the image file (example: '/data/user/0/.../image.jpg')
+    path, // the URI of the image (example: 'file:///data/user/0/.../image.jpg')
```

Comment:
According to the PR description - the `uri` fields have the `file://` scheme and `path` do not - here in the README is the other way around - is it mistake in README or in PR description?

```
// web
{
    "width": 1125,
    "height": 750,
    "name": "ReactNative_cropped_image.jpeg",
    "size": 283213,
    "path": "blob:http://localhost:8082/43ff7a16-c464-4130-a3d2-95286bae46b1",
    "uri": "data:image/jpeg;base64,/9j/4A|"
}
// android
{
  "height": 600,
  "size": 31520,
  "name": "ReactNative_cropped_image_5129169684629022727.jpg",
  "width": 800,
  "uri": "file:///data/user/0/com.rn73/cache/ReactNative_cropped_image_5129169684629022727.jpg",
  "path": "/data/user/0/com.rn73/cache/ReactNative_cropped_image_5129169684629022727.jpg"
}
// ios
{
  "path": "/Users/davydnarbutovich/Library/Developer/CoreSimulator/Devices/0CA37DFB-8AF0-4714-A849-82E2EE03634F/data/Containers/Data/Application/AD103DB7-10DA-4D4E-A7DD-BDCE371A961D/Library/Caches/ReactNative_cropped_image_/431C35F1-29EC-4972-84C5-01574DDCFDEE.png",
  "size": 280504,
  "height": 800,
  "name": "431C35F1-29EC-4972-84C5-01574DDCFDEE.png",
  "uri": "file:///Users/davydnarbutovich/Library/Developer/CoreSimulator/Devices/0CA37DFB-8AF0-4714-A849-82E2EE03634F/data/Containers/Data/Application/AD103DB7-10DA-4D4E-A7DD-BDCE371A961D/Library/Caches/ReactNative_cropped_image_/431C35F1-29EC-4972-84C5-01574DDCFDEE.png",
  "width": 600
}
```

Reply:
mistake in README)